### PR TITLE
Settings and initial contents for Portuguese localization

### PR DESCRIPTION
**Pull request by edsoncelio**
This PR adds the base configs to Portuguese localization.

Checklist
* [x] Add Portuguese config section to `/config.toml`
* [x] Add `content/pt/_index.md` in Portuguese
* [x] Add `content/pt/contribute/_index.md` in Portuguese (WIP)
* [x] Add `content/pt/style-guide/_index.md` in Portuguese (WIP)

**Preview** (**UPDATED**):

image

### CODE REVIEW

**Comment by edsoncelio** [comment]
Hi, @seokho-son thanks for the information! Right now I'm having some issues with i18n site strings to `pt`, in short: If we use the configuration just to `pt` I receive warnings and the strings localization doesn't work because the localization is using a Portuguese variation (Brazilian Portuguese), to fix we need to specify that we are using config to `pt-br`, I'll make more test and then I'll put to review :)

**Comment by seokho-son** [comment]
@edsoncelio oh, there is a country-specific variants issue.

I am not sure these links from K8s are helpful for you. 

https://github.com/kubernetes/website/blob/main/config.toml#L404

>[languages.pt-br]
title = "Kubernetes"
description = "Orquestração de contêineres em nível de produção"
languageName ="Português"
weight = 9
contentDir = "content/pt-br"
languagedirection = "ltr"

https://github.com/kubernetes/website/tree/main/content/pt-br

> content/pt-br/

https://github.com/kubernetes/website/blob/main/data/i18n/pt/pt-br.toml

> i18n/pt/pt-br.toml


By the way, what do you think so? Do we need to differentiate localization for Portuguese (Brazil) and Portuguese (Portugal) ? (team, members, branch, label, and so on)

**Comment by edsoncelio** [comment]
Thanks! I'm using the k8s config as a reference, we had the same issue and was decided to use `pt-br` instead of `pt`.
For this project I think we can proceed in this way too, all of the localization team are Brazilians, so my suggestion is to use the configs with `pt-br` and the labels/branch can continue as `pt`.

What you think @brunoguidone?

**Comment by brunoguidone** [comment]
Sure Edson, totally agree with you.

**Comment by edsoncelio** [comment]
@brunoguidone PTAL :)

File: content/pt-br/style-guide/_index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+- **O que é**: forneça uma visão curta e clara da definição abordada.
+- **Que problema isso resolve**: foque no problema e não na solução (que vem na seção seguinte), e se puder, não mencione o termo que está sendo definido.
+- **Como isso resolve o problema**: nessa seção volte ao termo e explique como ele resolve o problema da seção anterior.
+
+As definições de **propriedade** não requerem essas três seções separadas, apenas a definição do termo é suficiente.
+
+#### Busque manter as definições simples
+
+O glossário busca ***explicar conceitos complexos em poucas palavras**, o que torna uma tarefa muito difícil e que provavelmente vai exigir muitas revisões. Sempre tenha em mente
```
Comment (seokho-son) [inline content/pt-br/style-guide/_index.md:111]:
```suggestion
O glossário busca **explicar conceitos complexos em poucas palavras**, o que torna uma tarefa muito difícil e que provavelmente vai exigir muitas revisões. Sempre tenha em mente
```

File: config.toml [hunk 2]
```diff
@@ -66,6 +67,13 @@ weight = 1
 #time_format_default = "02.01.2006"
 #time_format_blog = "02.01.2006"
 
+[languages.pt-br]
+title = "Glossário Cloud Native"
+description = "O Projeto CNCF Glossário Cloud Native se destina a ser usado como uma referência para termos comuns usados ao falar sobre aplicações nativas de nuvem."
+languageName = "Português"
+contentDir =  "content/pt-br"
+weight = 1
```
Comment (seokho-son) [inline config.toml:75]:
```suggestion
weight = 2
```
We should not use the same weight with English.
(FYI: this weight will be adjusted when we merge dev-xx to main)

**Review by seokho-son — COMMENTED** [review]
Requires minor modifications :)

**Comment by edsoncelio** [comment]
> Requires minor modifications :)

Thanks, done!

**Comment by seokho-son** [comment]
@edsoncelio 
Do we need both i18n/**pt-br**.toml and i18n/**pt**.toml ?

**Comment by edsoncelio** [comment]
> @edsoncelio Do we need both i18n/**pt-br**.toml and i18n/**pt**.toml ?

No, removed :D

**Review by seokho-son — APPROVED** [review]
LGTM!!

(target branch is dev-pt)

- [x] 3-1. Adding a new language setting to site configuration https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-1-adding-a-new-language-setting-to-site-configuration
- [x] 3-2. Adding a file for site strings https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-2-adding-a-file-for-site-strings
- [x] 3-3. Adding a new localization directory https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-3-adding-a-new-localization-directory
- [x] 3-4. Minimum required content https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-4-minimum-required-content
- [x] 3-5. Check configuration works and open a PR https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-5-check-configuration-works-and-open-a-pr

**Comment by seokho-son** [comment]
PTAL @CathPag :)

**Review by seokho-son — APPROVED** [review]

**Pull request merged**